lib/Dialect/TTIR/IR/TTIROps.cpp

```diff
@@ -36,6 +36,26 @@ ::mlir::OpFoldResult mlir::tt::ttir::ConstantOp::fold(FoldAdaptor adaptor) {
   return getValueAttr();
 }
 
+::mlir::OpFoldResult mlir::tt::ttir::GetDimensionSizeOp::fold(FoldAdaptor adaptor) {
```

Comment:
This is the section for Constant op, and you are defining the folder for GetDimensionSizeOp. Can you move this peace of code to the section related to the GetDimensionSizeOp?